src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAjaxPageData.java

```diff
@@ -21,4 +22,39 @@
     public InstructorFeedbackResponseCommentAjaxPageData(AccountAttributes account) {
         super(account);
     }
+
+    public FeedbackResponseComment getComment() {
+        FeedbackResponseComment frc = new FeedbackResponseComment(comment, comment.giverEmail);
+
+        frc.setResponseVisibleToRecipient(true);
+        frc.setResponseVisibleToGiverTeam(true);
+        frc.setResponseVisibleToRecipientTeam(true);
+        frc.setResponseVisibleToStudents(true);
+        frc.setResponseVisibleToInstructors(true);
+        frc.setEditDeleteEnabled(true);
+        frc.setInstructorAllowedToDelete(true);
+        frc.setInstructorAllowedToEdit(true);
+        frc.setResponseGiverName(giverName);
+        frc.setResponseRecipientName(recipientName);
+        frc.setShowCommentToString(showCommentToString);
+        frc.setShowGiverNameToString(showGiverNameToString);
```

Comment:
Maybe it'll be better to create a method / constructor in `FeedbackResponseComment` for this?
There could be certain default values in there, like for those booleans.